packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -229,7 +229,6 @@ const DatePickerInput = (props) => {
 
   const additionalInputProps = { ...customProps, ...inputAttributes };
   const momentDateFormat = useMemo(() => DateUtil.getFormatByLocale(intl.locale), [intl.locale]);
-  const nameLabelId = `name-label-${uuidv4()}`;
```

Comment:
This `Id` was created to fix the issue of label not being announced by screen reader, Does this PR still ensures label announcement by screen readers..?

Reply:
This Id was tagged to name attribute. And even earlier visual label is announced by the screen reader. Ashish has created a PR for the label announcement.